source/grid/grid_tools.cc

```diff
@@ -859,13 +859,14 @@ namespace GridTools
 
 
   template <int dim, int spacedim>
-  void
-  invert_all_negative_measure_cells(
+  std::size_t
+  invert_cells_with_negative_measure(
     const std::vector<Point<spacedim>> &all_vertices,
     std::vector<CellData<dim>> &        cells)
   {
+    // This function will always throw an exception outside of codimension 0
```

Comment:
This comment should be in the function documentation in the .h file instead of here, so that it shows up in the online documentation.